### ISSUE TITLE
dep issue

### ISSUE THREAD

**Issue by keredson** [keredson/bottle-react#6]
if you have a file like this:
```jsx
var WhatEv = React.createClass({
  render: function() {
    return(<Part />);
  },
})
bottlereact._register('WhatEv', WhatEv)

var Part = React.createClass({
  render: function() {
    return(<div>part</div>);
  },
})
```

BR sometimes can try to render the page before `Part` has been defined.  This is because `register` can initiate a render if it's the last dependency met.

**Comment by keredson** [comment]
to fix this, trigger the render w/ a `setTimeout` of 0.  this will put it next in the queue, rather than before the current file has finished parsing.

**Comment by keredson** [comment]
fixed in `0.4.3`
